src/main/java/com/jmb/springfactory/model/factory/RolDtoFactory.java

```diff
@@ -0,0 +1,31 @@
+package com.jmb.springfactory.model.factory;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.jmb.springfactory.model.dto.RolDto;
+
+public class RolDtoFactory {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)